pkg/tasks/python.go

```diff
@@ -27,13 +28,54 @@ func parserPython(config *taskConfig, task *Task) error {
 	task.featureName = "python"
 	task.featureParam = version
 
+	task.addAction(&pyenvPath{})
+	task.addAction(&pyenv{})
 	task.addAction(&pythonPyenv{version: version})
 	task.addAction(&pythonInstallVenv{version: version})
 	task.addAction(&pythonCreateVenv{version: version})
 
 	return nil
 }
 
+type pyenvPath struct{}
+
+func (p *pyenvPath) description() string {
+	return ""
+}
+
+func (p *pyenvPath) needed(ctx *context) (bool, error) {
+	match, _ := regexp.MatchString("\\.pyenv/bin", ctx.env.Get("PATH"))
+	return !match, nil
+}
+
+func (p *pyenvPath) run(ctx *context) error {
+	ctx.ui.TaskWarning("The PATH environment variable should have 'pyenv' in it. Try adding the following to ~/.bashrc\n\nexport PATH=\"~/.pyenv/bin:$PATH\"")
+	return nil
+}
+
+type pyenv struct{}
+
+func (p *pyenv) description() string {
+	return "install PyEnv"
+}
+
+func (p *pyenv) needed(ctx *context) (bool, error) {
+	_, err := helpers.NewPyEnv()
+	if err == nil {
+		return false, nil
+	}
+
+	return true, err
+}
+
+func (p *pyenv) run(ctx *context) error {
+	err := command(ctx, "bash", "-c", "curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash").Run()
```

Comment:
This is hacky and should probably be done another way. I wanted to try and add a homebrew task, but wasn't sure how to reach out to do that. At the very least should probably create separate commands that are piped together.